locals.tf

```diff
@@ -26,17 +26,14 @@ locals {
 # task_definition map
 locals {
   container_definition_file = "${lookup(var.task_definition, "container_definition_file", "containers.json")}"
-  cpu                       = "${lookup(var.task_definition, "cpu", var.launch_type == "FARGATE" ? "256" : "")}"
-  memory                    = "${lookup(var.task_definition, "memory", var.launch_type == "FARGATE" ? "512" : "0")}"
+  cpu                       = "${lookup(var.task_definition, "cpu", "256")}"
+  memory                    = "${lookup(var.task_definition, "memory", "512")}"
```

Comment:
Shouldn't the `cpu` and `memory` values be numbers instead of strings?

Reply:
This is old 0.11 code. There should be no changes whatsoever in the *.tf files. The point of the original pull request was to simplly add a test directory with tests for 0.11.  Unfortunately, mystery code changes were added with the test code. No idea where they came from. Neither does Evan. This pull request is to reversing the changes made, and make some minor changes to the test code.